### ISSUE TITLE
Display more info about validators

### ISSUE THREAD

**Issue by fedekunze** [cosmos/voyager#1100]
UI Version: 0.9.3

Description:

Would be a nice to have to display the same information as the explorer as well as other important information **within** the Validator Profile:

|   Validator Struct   | Explorer    | Voyager |
| --------|---------|-------|
| `Owner`  |  ✅  | ⛔️ |
| `PubKey` | ✅ | ⛔️ |
| `Revoked`  |  ✅  | ⛔️ |
| `Status` | ✅ | ⛔️ |
| `Tokens`  |  ⛔️  | ⛔️ |
| `DelegatorShares` | ⛔️ | ⛔️ |
| `Moniker` | ✅ | ✅ |
| `Identity`  |  ✅  | ⛔️ |
| `Website` | ✅ | ✅ |
| `Details` | ✅ | ✅ |
| `BondHeight`  |  ✅  | ⛔️ |
| `BondIntraTxCounter` | ⛔️ | ⛔️ |
| `ProposerRewardPool`  | ⛔️  | ⛔️ |
| `Commission` | ⛔️ | ⛔️ |
| `CommissionMax`  |  ⛔️  | ⛔️ |
| `DelegatorShares` | ⛔️ | ⛔️ |
| `CommissionChangeRate` | ⛔️ | ⛔️ |
| `CommissionChangeRateToday`  |  ⛔️  | ⛔️ |
| `LastBondedTokens` | ⛔️ | ⛔️ |

Also both of them have **Total Vote Power**. It would be nice to also show the stake and delegator shares as well as the total vote power

**Comment by jbibla** [comment]
thanks @fedekunze this is super helpful. we should actually share this component in tendermint/ui...

**Comment by NodeGuy** [comment]
Nice table!

**Comment by jbibla** [comment]
yes, because the explorer has a validator list and profile page.

**Comment by NodeGuy** [comment]
I created a new issue for that: https://github.com/cosmos/voyager/issues/1152

**Comment by NodeGuy** [comment]
Blocked by https://github.com/cosmos/voyager/issues/1154.

**Comment by faboweb** [comment]
Proposal:
![image](https://user-images.githubusercontent.com/5869273/44338462-3b264b00-a47e-11e8-87e7-ff7de1751a23.png)
![image](https://user-images.githubusercontent.com/5869273/44338478-45484980-a47e-11e8-9d26-0d31adff3b19.png)

I converted `DelegatorShares` into `Self Bond` as I think this is the actual interesting value.
Do we need `BondIntraTxCounter` (what is that?) and `ProposerRewardPool`? I think those are only used internally together with `LastBondedTokens`. Correct me if I am wrong.

Pulling in @fedekunze

**Comment by faboweb** [comment]
@nylira what about a default avatar?

**Comment by fedekunze** [comment]
@faboweb Imo `LastBondedTokens` are useful. They are printed on the SDK as _Previous Bonded Tokens:_ `LastBondedTokens`. Same with `ProposerRewardPool` and `CommissionChangeToday`. We can leave `BondIntraTxCounter` out 

- `ProposerRewardPool` : the reward pool collected from being the proposer
- `BondIntraTxCounter`: block-local tx index of validator change

**Comment by faboweb** [comment]
How would you base decisions on any of those 3 values?

**Comment by jbibla** [comment]
> what about a default avatar?

please use this icon: https://drive.google.com/open?id=11_B4Q5SsErJ4G5v2VermpCbumYX2wU-U

**Comment by faboweb** [comment]
Updated the screenshots.

**Comment by fedekunze** [comment]
closed by #1164
